lib/internal/util.js

```diff
@@ -10,7 +18,7 @@ exports.printDeprecationMessage = function(msg, warned) {
   if (process.throwDeprecation)
     throw new Error(msg);
   else if (process.traceDeprecation)
-    console.trace(msg);
+    console.trace(msg.replace(new RegExp(`^\\(${prefix}\\) `), ''));
```

Comment:
Oh, right. Template string substitution won't work inside a regex literal, so this is fine.